README.md

```diff
@@ -1,93 +0,0 @@
-![NLog](https://raw.githubusercontent.com/NLog/NLog.github.io/master/images/NLog-logo-only_small.png)
```

Comment:
Oops didn't see that this pull-requests also removes the main-page for the entire NLog.Web-repo. Please update the pull-request to not delete the file:

- https://github.com/NLog/NLog.Web/blob/master/README.md

Reply:
Certainly

Reply:
Completed